### DISCUSSION TITLE
Search tickets by using contents of related emails as search terms

### DISCUSSION THREAD

**Discussion by afatac** [YetiForceCompany/YetiForceCRM#17025] [General]
Please see the screenshot below.

![image](https://user-images.githubusercontent.com/26587885/231967394-da13db12-49d1-48ca-9dd2-ac69cd0d9daa.png)

I tried searching for tickets using some words in the body messages of related emails as search terms. However, no search result is returned.

Is the related email indexed for searching?

**Comment by mariuszkrzaczkowski** [comment]
https://gitdeveloper.yetiforce.com/index.php?module=Search&parent=Settings&view=Index

![image](https://user-images.githubusercontent.com/10198654/231971758-389c54b1-750a-40fd-917b-e2e9dc4f4e00.png)

**Reply by afatac** [reply]
OK... we can add more fields for searching. But there is no field that can help in searching the related email content.

The Description field only contains content of first email
